### DISCUSSION TITLE
Allow open branches for members of thin-edge

### DISCUSSION THREAD

**Discussion by mbay-ODW** [thin-edge/thin-edge.io_examples#25] [General]
@AndrejS321 and Team:
We started contributing to the thin-edge.io_example repo with various extensions. However currently we have to fork the repo first, do our changes and create a pull request based on the fork. Could we not just allow members of that organization to create own branches such that we can create PR´s from there?

I know its not a huge problem do it like that but the additional step is annoying.

**Comment by reubenmiller** [comment]
This is a really old post, however using the fork model is the best way to coordinate changes over multiple people and reduce the risk of someone accidentally doing something they shouldn't (e.g. force push over someone else's branch).

For thin-edge.io we use this model and haven't had any problems with it, and you get used to fast forwarding your fork to keep up with upstream (it only takes one click in the ui).

So closing this as using forks should be the default way of doing things.
